code/modules/antagonists/valentines/heartbreaker.dm

```diff
@@ -6,19 +6,43 @@
 
 
 /datum/antagonist/heartbreaker/proc/forge_objectives()
-	var/datum/objective/martyr/normiesgetout = new
-	normiesgetout.owner = owner
-	objectives += normiesgetout
-	log_objective(owner, normiesgetout.explanation_text)
+	var/datum/objective/O = new
+	O.explanation_text = "Ruin people's dates through non-lethal means."
+	O.owner = owner
+	O.completed = TRUE // no good way to check this
+	objectives += O
+	log_objective(owner, O.explanation_text)
+	if(prob(30)) // rare chance to get martyr, really ruin those dates!
+		O.explanation_text = "Ruin people's dates however necessary."
+		var/datum/objective/martyr/normiesgetout = new
+		normiesgetout.owner = owner
+		objectives += normiesgetout
+		log_objective(owner, normiesgetout.explanation_text)
```

Comment:
Use a subtype for the objective explanation text, explanation texts get overwritten.

Reply:
Custom objectives are always done this way >:( it's not worth it but ok

Reply:
(it doesn't get overwritten)